addons/stock/models/stock_production_lot.py

```diff
@@ -11,6 +11,7 @@ class ProductionLot(models.Model):
     _inherit = ['mail.thread','mail.activity.mixin']
     _description = 'Lot/Serial'
     _check_company_auto = True
+    _order = 'name'
```

Comment:
I would add the `id` to ensure a deterministic behavior 
```suggestion
    _order = 'name, id'
```
Otherwise, lgtm